public/locales/i18next-parser.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   // Default namespace used in your i18next config
   defaultNamespace: 'grafana',
-
-  locales: ['en-US', 'fr-FR', 'es-ES', "de-DE", "zh-Hans", 'pseudo-LOCALE'],
+  // Adds changes only to en-US and pseudo-LOCALE when extracting keys, every other language is provided by Crowdin
+  locales: ['en-US', 'pseudo-LOCALE'],
```

Comment:
Yes. In fact, I don't think it's even necessary to have `pseudo-LOCALE` in here either because the pseudo script will overwrite whatever's in it anyway from the english file.

Reply:
if we don't need pseudo, we should probably remove it then 👍